modin/experimental/engines/omnisci_on_ray/test/test_dataframe.py

```diff
@@ -1420,6 +1420,31 @@ def merge(df1, df2, **kwargs):
 
         run_and_compare(merge, data=self.dt_data1, data2=self.dt_data2)
 
+    left_data = {"a": [1, 2, 3, 4], "b": [10, 20, 30, 40], "c": [11, 12, 13, 14]}
+    right_data = {"c": [1, 2, 3, 4], "b": [10, 20, 30, 40], "d": [100, 200, 300, 400]}
+
+    @pytest.mark.parametrize(
+        "left_on, right_on", [["a", "c"], [["a", "b"], ["c", "b"]]]
+    )
+    def test_merge_left_right_on(self, left_on, right_on):
+        def merge(df1, df2, left_on, right_on, **kwargs):
+            return df1.merge(df2, how="left", left_on=left_on, right_on=right_on)
```

Comment:
Let's add a case with "inner" type of join to make sure the changes work for it too.

Reply:
Done

Reply:
Thanks!